empire.go

```diff
@@ -541,6 +542,9 @@ type DeployOpts struct {
 
 	// Commit message
 	Message string
+
+	// Updates is a stream to publish status updates to during deploy
+	Updates status.StatusStream
```

Comment:
Since we're only using this to write status updates to the io.Writer, we might as well just remove this option and move status.NewStatusStream() to [here](https://github.com/mhahn/empire/blob/f3afbd7e24e98c275d932581fc1c9afe31c6cb32/deployments.go#L71)